docker/test/integration/resources/minifi_cpp_with_provenance_repo/minifi.properties

```diff
@@ -0,0 +1,32 @@
+nifi.version=0.6.0
```

Comment:
also not important at all, but the version should be `0.12.0`

Reply:
Done.